README.md

```diff
@@ -1,31 +1,66 @@
+and production.
+* [Deploy to Kubernetes with Helm](docs/helm.md) - If you've built a Helm chart for your application,
+rok8s-scripts is a great way to deploy your chart to staging and production
+* [Manage secrets](docs/secrets.md) - Learn how rok8s-scripts can simplify and secure your secret management workflows
 
-The latest Debian Stretch release can be pulled from `quay.io/reactiveops/ci-images:v8-stretch`. A full list of the latest image tags is available on our [Quay repository](https://quay.io/repository/reactiveops/ci-images).
+### Cloud-specific Documentation
+* [Deploy to AWS](docs/aws.md) - Learn how to authenticate and deploy using rok8s-scripts with aws-cli
+* [Deploy to GCP](docs/gcp.md) - Learn how to authenticate and deploy using rok8s-scripts with gcloud
 
 ## Examples
 
-rok8s-scripts is designed to work well in a wide variety of environments. That includes Bitbucket Pipelines, CircleCI, GitLab CI, and more. There are many valid ways to configure CI pipelines, we've includes a variety of [examples](/examples) in this repository.
-
-Most notably, the CI example includes sample configuration for the following platforms:
+rok8s-scripts is designed to work well with a wide variety of use cases and environments.
+There are many valid ways to configure CI pipelines, but to help you get started, we've included a variety of [examples](/examples) in this repository.
 
+### CI Platforms
 - [Bitbucket Pipelines](/examples/ci/bitbucket-pipelines.yml)
 - [CircleCI](/examples/ci/.circleci/config.yml)
 - [GitLab CI](/examples/ci/.gitlab-ci.yml)
 - [Jenkins](/examples/ci/Jenkinsfile)
 
-On their own, these examples may not make a lot of sense. There's a lot more documentation below that should cover everything included in these examples and more.
+### Miscellaneous examples
+- [External secrets manager](/examples/external-secrets-manager)
+- [SOPS secrets](/examples/sops-secrets) - Shows how to use [sops](https://github.com/mozilla/sops) with rok8s-scripts
+- [Using Helm](/examples/helm) - We recommend using Helm to manage your deployments
+- [Optional components](/examples/optional-components) - Turn components (e.g. Horizontal Pod Audoscaler) on and off depending on whether you're deploying to staging or production
+- [Production ready](/examples/production-ready) - Includes a number of recommended production features
+
+## CI Images
+
+Each new release of rok8s-scripts comes with a set of CI images for simple workflows. These images include a set of common CI/CD dependencies, including Docker, Kubernetes, Helm, AWS, and Google Cloud client libraries. Starting with these images as a base for deployment workflows ensures that you don't need to spend any build time installing extra dependencies.
+
+We currently include a variety of CI Images, including Alpine and Debian Stretch as our recommended starting points. In certain cases you may want to use our images that include Node.js or Golang.
```

Comment:
Can we remove the references to the language-specific images?  Our goal is to remove them and I think that this might be a good starting place for that.

Reply:
Sounds good.